manager.go

```diff
@@ -638,23 +638,31 @@ func (m *jobManager) resolveImageOrVersion(imageOrVersion, defaultImageOrVersion
 		return unresolved, "", nil
 	}
 
-	is, err := m.imageClient.ImageV1().ImageStreams("ocp").Get("release", metav1.GetOptions{})
```

Comment:
I would suggest instead changing to a for loop where you walk over `[]string{"ocp","origin"}` and search each dir.  If no match, exit.  You might want to extract the code below into a function that takes an image stream and then invoke the method in a loop.

Reply:
Build the URL based on the image stream name you find (switch statement), move that into it's own method.

Reply:
Updated PTAL. Didn't move this to the function yet so that #81 would not cause too many merge conflicts